lib/Zonemaster/Backend/TestAgent.pm

```diff
@@ -58,7 +58,7 @@ sub run {
     }
     $domain = $self->to_idn( $domain );
 
-    if (defined $params->{ipv4} || defined $params->{ipv4}) {
+    if (defined $params->{ipv4} || defined $params->{ipv6}) {
```

Comment:
Could you update this file to be consistent about spaces and tabs in the indentation?

Reply:
Yeah, I made a mistake by forking from a previous issue. As it is a very small code piece I'm keeping it to avoid having to merge.

Reply:
Please do rebase to remove overlapping with other PRs.